Makefiles/db.toml

```diff
@@ -17,7 +17,7 @@ category = "db"
 script_runner = "@shell"
 script = '''
 cd ${CARGO_MAKE_WORKSPACE_WORKING_DIRECTORY}/src/event-db
-psql -e postgres -f setup/dev-db.sql ${@}
+psql -e -U postgres -f setup/dev-db.sql ${@}
```

Comment:
better not to add this option here. As we are using ${@} argument we can pass it like this
```
cargo make local-event-db-init -U postgres
```

Reply:
We are looking at improving CI/CD let's fix this issue when we do that.
The `-U` fixes the original file because it's broken without it.